src/hotspot/share/gc/shared/gc_globals.hpp

```diff
@@ -131,6 +131,12 @@
   product(bool, UseShenandoahGC, false,                                     \
           "Use the Shenandoah garbage collector")                           \
                                                                             \
+  product(ccstr, ErgonomicsProfile, "shared",                               \
+          "Ergonomics profile to use. "                                     \
+          "\"shared\" (default) for when the JVM is running "         \
```

Comment:
The continuation back slashes seem to have the wrong whitespace

Reply:
It will be the default if `is_containerized` returns true. We are aware of JDK-8261242, but we believe that it is an improvement to is_containerized, not necessarily a bug. Then again, in most cases, it will return the correct info. In the cases where it does not, users can still override the profile with `-XX:ErgonomicsProfile=shared`.